cloudhub/1.0/modules/ROOT/pages/cloudhub-connector-ref.adoc

```diff
@@ -210,7 +210,7 @@ Lists all the available notifications for a giver domain.
 | Page Size a| Number |  Size of the page to retrieve per iteration. This only should be changed for performance purposes. |  25 |
 | Streaming Strategy a| * <<repeatable-in-memory-iterable>>
 * <<repeatable-file-store-iterable>>
-* <<non-repeatable-iterable>> |  Configure if repeatable streams should be used and their behavior |  |
+* Non repeatable iterable |  Configure if repeatable streams should be used and their behavior |  |
```

Comment:
This field description should be improved at some point. Maybe make a note of this? The Connector Reference Field Descriptions spreadsheet has this description for Streaming Strategy:

Configures how Mule processes streams. Repeatable streams are the default behavior.

It's also strange that one of the options has a description and the others don't. If you look at the docs for Salesforce Connector, you will see the descriptions in the Types section.